warehouse/models/staging/gtfs_guidelines/stg_gtfs_guidelines__no_rt_critical_validation_errors.sql

```diff
@@ -0,0 +1,67 @@
+WITH feed_guideline_index AS (
+    SELECT * FROM {{ ref('stg_gtfs_guidelines__feed_guideline_index') }}
+    WHERE check = {{ no_rt_critical_validation_errors() }}
+),
+
+-- gtfs_rt_fact_files_wide_hourly has one row per day per ID+URL
+gtfs_rt_fact_files_wide_daily AS (
+SELECT * FROM {{ ref('gtfs_rt_fact_files_wide_hourly') }}
+),
+
+gtfs_rt_fact_daily_validation_errors AS (
+    SELECT * FROM {{ ref('gtfs_rt_fact_daily_validation_errors') }}
+),
+
+gtfs_rt_validation_code_descriptions AS (
+    SELECT * FROM {{ ref('gtfs_rt_validation_code_descriptions') }}
+),
+
+rt_files_by_day AS (
+    SELECT
+        calitp_itp_id,
+        calitp_url_number,
+        date_extracted AS date,
+        COUNT(*) AS rt_files
```

Comment:
Just to confirm, I think that this is just going to return the count of the rows, so it will either be 1, 2, or 3. If you want the actual count of files, I think you'd want to do `SUM(file_count_day)`. For these purposes it works out to the same thing because a row will only be present if there was at least one file, but just wanted to confirm (if we were to ever change the implementation of this table to have a row per URL even if there was no data, that could change.)

Reply:
Yep, this is my understanding - thinking that they get credit if they have at least one RT file present on a given day (I will also review the business logic with my peers). My thinking is that, since we will also score them on:
a) presence of each file
b) updates every 20sec for vehicle positions & trip updates
...it's OK to be lax on this check wrt what it means for an RT file to be present